src/interfaces.ts

```diff
@@ -72,6 +72,15 @@ export interface ILoggerConfig {
    * in place of the console, but still alongside this logger.
    */
   logger?: ILogger;
+  /**
+   * These are essentially interceptors for log messages. They will allow
+   * you to change the level, message, details or log options for any given
+   * message. There are three options for handling messages:
+   *   next() - sends message as it was received to the next formatter
+   *   next(level, message, details, options) - sends message to the next formatter with the specified params
+   *   not calling next() at all - don't log the message
```

Comment:
I know that this isn't advertised for customers to consume, so I'm not super concerned with these docs being really nice. But it may be nice (for our sakes') to add a few things:

* update the CHANGELOG.md
* Should you add any info about what the default formatter is? 
* This config option needs to be added to the README. (This repo just copy/pastes what is in here – no fancy stuff like the SDK)
* Maybe add an example of a formatter to the README as well.

Reply:
thanks for the reminder. Initially the goal of this was just to see if anyone noticed any pitfalls to this approach. I'll work on putting the final touches on now.